### PR TITLE
remove dependency on forked go-kzg

### PR DESCRIPTION
**Pull request by roberto-bayardo**
removed go.mod pointers to the forked repo, ran go mod tidy, updated submodules to latest commit which also removes those dependencies on forked go-kzg

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,14 +3,11 @@ module github.com/Inphi/eip4844-interop
 go 1.18
 
 require (
-	github.com/ethereum-optimism/optimism/bss-core v0.0.0-20221007221122-8c8adcf67f91
```
Comment (Inphi) [inline go.mod:6]: hmm. is this a conflict. Dont remember adding this here before.

**Comment by roberto-bayardo** [comment]
Actually hold off a bit let me update the subrepos to also use the EvaluatePolyInEvaluationForm which is now in go-kzg.

**Comment by roberto-bayardo** [comment]
Ok this is updated & should be ready to go now.

File: go.mod (hunk 1 above)
Comment (roberto-bayardo) [reply go.mod:6]: I don't know where this came from, but looks like go mod tidy figured out it wasn't needed?

Comment (roberto-bayardo) [reply go.mod:6]: here's the commit that added it: https://github.com/Inphi/eip4844-interop/commit/5f878dfc3318abdc6acc92a9117d65889d4db7bb

Comment (roberto-bayardo) [reply go.mod:6]: @mdehoog says it's from some unchecked in code from his repo, so it's not needed.

**Review by Inphi — COMMENTED** [review]
Looks good. Do you still need to update the prysm submodules for the rootsOfUnity bugfix?

nvmind. I see that tests are passing now 🎉

**Review by Inphi — APPROVED** [review]

**Pull request merged**